datacontract/cli.py

```diff
@@ -211,6 +211,7 @@ class ImportFormat(str, Enum):
     jsonschema = "jsonschema"
     odcs = "odcs"
     unity = "unity"
+    spark = "spark"
```

Comment:
This importer format seems wrong here (and causes a merge conflict)

Reply:
I resolved conflict.
By the way it was a mistake to put spark in `ImportFormat`